lib/blockstore/README.md

```diff
@@ -0,0 +1,147 @@
+# BlockStore
+
```

Comment:
Can you add a comment in here about the atomicity of the `BlockStore`?

Reply:
Hmm... the latest atomicity fixes are for chaindb, comments have been added to that code that the blocks are removed _after_ successful commit to chaindb.